packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -622,7 +622,9 @@ LightningElement.prototype = {
         return vm.shadowRoot;
     },
 
-    get refs(): RefNodes | undefined {
+    // TODO: fix?
```

Comment:
Reading through the code here, it seems like `undefined` is a valid return value, so I think we should update the base type to indicate that. But that seems likely to have a ripple effect downstream!